### ISSUE TITLE
Create container image for Apple Silicon M1/M2

### ISSUE THREAD

**Issue by hguerrero** [crc-org/crc-cloud#116]
Current container image is linux/x86_64. An new image is required for Apple Mac with Silicon M1/M2 processors.

**Comment by praveenkumar** [comment]
@hguerrero we only created linux images to consume this project as part of CI, what is the requirement for having it for M1/M2?

**Comment by gbraad** [comment]
@adrianriobo @praveenkumar where is the actual Dockerfile ? IT does not seem to be part of this repo

**Comment by adrianriobo** [comment]
It is located under `oci` folder https://github.com/crc-org/crc-cloud/blob/main/oci/Containerfile

**Comment by cfergeau** [comment]
On M1/M2, I'd recommend using https://github.com/crc-org/crc directly. Or do you have a very different usecase?

**Comment by gbraad** [comment]
I believe he wants to trigger the job from his local machine, though it might be helpful to just set up a coordination VM

**Comment by adrianriobo** [comment]
> coordination VM

**Comment by gbraad** [comment]
right, but I see this as a low priority. it is more like a 'nice to have' than a hard requirement.

**Comment by adrianriobo** [comment]
yeah but as you said even if he wants to contribute on that regard that would be great :)

**Comment by hguerrero** [comment]
For a simpler starting experience, it is simpler to merely construct a container image that is compatible with M1/M2 due to the requirements to install Pulumi and other components. For demonstration purposes, I only need to manually trigger an instance in my situation.
